packages/platform-server/src/utils.ts

```diff
@@ -210,18 +210,20 @@ async function _render(platformRef: PlatformRef, applicationRef: ApplicationRef)
   }
 
   appendServerContextInfo(applicationRef);
-  const output = platformState.renderToString();
+  return platformState.renderToString();
+}
 
-  // Destroy the application in a macrotask, this allows pending promises to be settled and errors
-  // to be surfaced to the users.
+/**
+ * Destroy the application in a macrotask, this allows pending promises to be settled and errors
+ * to be surfaced to the users.
+ */
+async function asyncDestroyPlatform(platformRef: PlatformRef): Promise<void> {
   await new Promise<void>((resolve) => {
```

Comment:
Avoid creating an extra microtask.

```suggestion
function asyncDestroyPlatform(platformRef: PlatformRef): Promise<void> {
  return new Promise<void>((resolve) => {
```

Reply:
Thank you. done